src/js/server/schemaHandler.js

```diff
@@ -0,0 +1,54 @@
+/*
+
+    An extension of `gpii.express.handler` that adds JSON Schema headers to the outgoing response.
+
+    See this component's documentation for more details:
+
+    https://github.com/the-t-in-rtf/gpii-json-schema/blob/GPII-1336/docs/handler.md
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-express");
+require("../common/hasRequiredOptions");
+
+fluid.registerNamespace("gpii.schema.handler");
+
+var quotedPrintable = require("quoted-printable");
+
+/**
+ *
+ * Send the appropriate headers and then let the underlying grade's `sendResponse` function take over.
+ *
+ * @param that - The handler component itself.
+ * @param response {Object} - The Express `response` object: http://expressjs.com/en/api.html#res
+ * @param statusCode - The numeric HTTP status code.
+ * @param body {Object} - The response payload to send to the browser.
+ */
+gpii.schema.handler.sendResponse = function (that, response, statusCode, body) {
```

Comment:
I was just thinking about this, the problem comes when we use something like `schemaMiddleware`, which needs to send a response that has the headers set for an error message, and not the headers that would be associated with a successful REST request.  It would either need to call the functions provided here on its own (and duplicate some of the invoker wiring), or would need to pass the request down to its own router that uses the relevant middleware.